packages/cli/src/bin/options.js

```diff
@@ -16,6 +16,7 @@ program.Command.prototype.withNetworkOptions = function () {
 program.Command.prototype.withPushOptions = function () {
   return this
     .option('--push [network]', 'push all changes to the specified network')
+    .option('--skip-compile', 'skips contract compilation')
```

Comment:
Note that `add` already has a skip-compile option, and adding it here as well caused the option to be listed twice on `add --help`. We should remove it from the add command.